.travis.yml

```diff
@@ -1,16 +1,19 @@
 language: python
+python:
+  - "2.7_with_system_site_packages"
+sudo: false
+addons:
+  apt:
+    sources:
+      - george-edison55-precise-backports
+    packages:
+      - libwww-perl
+      - libcss-dom-perl
+      - python-lxml
+      - cmake
+      - cmake-data
 install:
- - git clone https://github.com/w3c/respec.git
- - git clone https://github.com/dontcallmedom/webidl-checker.git
- - git clone https://github.com/dontcallmedom/widlproc
-# - git clone https://github.com/halindrome/linkchecker.git
- - cd widlproc && make obj/widlproc && cd ..
- - pip install html5lib lxml html5validator
- - sudo apt-get install libwww-perl libcss-dom-perl
+ - git clone https://github.com/w3c/webrtc-respec-ci.git
```

Comment:
Have you considered adopting a `git submodules` approach here?  I note that you have a similar problem for the respec-ci repo as well.  Using the latest is fine, but is this going to be stable?

Also, @dontcallmedom, you disabled issues on the other repo.  I was going to report some things for which I don't have the time to generate a PR.